core/driver/sources/src/main/scala/org/virtuslab/ideprobe/IdeProbePaths.scala

```diff
@@ -11,7 +11,7 @@ object IdeProbePaths {
   }
 
   def from(config: PathsConfig): IdeProbePaths = {
-    val basePath = config.base.getOrElse(Paths.get(System.getProperty("java.io.tmpdir")))
+    val basePath = config.base.getOrElse(Paths.get(System.getProperty("java.io.tmpdir") + "/ide_probe"))
```

Comment:
`Paths.get(System.getProperty("java.io.tmpdir")).resolve("ide-probe")` would be more clean. Also I meant `-` as in repo name